config/environments/development.rb

```diff
@@ -36,6 +36,8 @@
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 }
```

Comment:
magic! With mailcatcher installed we can now visit `localhost:1080` and see the mail which was sent from test runs on local host.